src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Tree/Dependencies/CrossTarget/ICrossTargetRuleHandler.cs

```diff
@@ -43,14 +43,5 @@ internal interface ICrossTargetRuleHandler<T> where T : IRuleChangeContext
                          ITargetedProjectContext targetedProjectContext,
                          bool isActiveContext,
                          T ruleChangeContext);
-
-        /// <summary>
-        /// Handles clearing any state specific to the given context being released.
-        /// </summary>
-        /// <param name="context">
-        ///     A <see cref="ITargetedProjectContext"/> being released.
-        /// </param>
-        Task OnContextReleasedAsync(ITargetedProjectContext context);
-
```

Comment:
Were there any implementations that weren't effectively no-ops?

Reply:
No, other than delegation on the the "nested" ones.